README.md

```diff
@@ -292,8 +287,23 @@ Also, make sure `ENABLE_ALLAUTH_SOCIAL_LOGIN = True` is present on your settings
 
 See the guide: [How to deploy your own instance](deployment-notes.md)
 
-**NOTE:** Before using in a public-facing environment, don't forget to change the default credentials! They're in
-`variables.env`, `sfs/settings/local.py` and `.gitlab_ci.yml`
+### Ansible
+
+We have a rudimentary ansible setup. To setup:
+
+1. Acquire ssh access into a machine.
+1. Edit [inventory.yaml](./ansible/inventory.yaml) with your machine's IP.
+1. Edit [vars.yaml](./ansible/vars.yaml) with your variables.
+1. Run the playbook with:
+
+```sh
+cd ansible/
+ansible-playbook playbook.yaml -v -e @vars.yaml
+```
+
+You can also run the following as an ssh check into your machine:
 
-**NOTE:** DON'T FORGET TO CHANGE FROM THE DEFAULTS IN THE PROD SERVER BEFORE RELEASE, BECAUSE THE CURRENT ONES ARE IN
-THE GIT HISTORY
+```sh
+ansible all -i inventory.yaml -m ping
+ansible-inventory -i inventory.yaml --list
```

Comment:
`inventory` is specified in `ansible.cfg` now, so doesn't need to be here too :)